documentation/tutorials/getting-started-with-ash-json-api.md

```diff
@@ -10,6 +10,18 @@
 mix igniter.install ash_json_api
 ```
 
+After running the command above, you might encounter an error:
+
+```
+** (UndefinedFunctionError) function Igniter.Libs.Phoenix.web_module_name/1 is undefined (module Igniter.Libs.Phoenix is not available)
+```
+
+This can be fixed by running:
+
+```
+mix deps.compile
+```
+
```

Comment:
I had this error when running `mix igniter.install ash_json_api`, but I could solve it with `mix deps.compile`. I am not sure if it is fine to add it here.

Reply:
This should be fixed in the latest version of igniter.

Reply:
I have removed this part. But FYI, I used the latest version of `ash_json_api` and I had this error.

Reply:
It was fixed in the latest `igniter` version, so to get that fix you'd have to upgrade `igniter`.